lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -322,7 +323,7 @@ sub _schedule_iso {
     }
     catch {
         my $error = shift;
-        OpenQA::Utils::log_warning("Failed to schedule ISO: $error");
+        push(@notes, "Transaction failed: $error");
```

Comment:
This is one of the places where scheduling an Iso could fail if there are loops. I think the previous message is clearer.

Reply:
This message is now shown in the scheduled products table (e.g. https://openqa.suse.de/admin/productlog). So it is clear that the failure is about scheduling an ISO.

Since it is also clear to which certain scheduling attempt the messages added here belong the timestamp is no loss (the scheduled product has a timestamp).

Reply:
Ahh cool then :)

Reply:
This let me thing that the table could still be improved (e.g. rows with failures should be highlighted). But I leave that for another PR.